o11nplugin-vro-package/src/main/resources/ScriptModule/com/vmware/actions/fetchObjectLists.xml

```diff
@@ -110,8 +110,21 @@
 				}
 				System.log("Port : "+Port)
 				return Port;
+			}else if(field == "healthmonitor"){
+				var poolUrl = object.pool_ref;
+				var path = poolUrl.split("/");
+				var response = aviPlugin.getObjectDataByUUID(path[path.length-2],path[path.length-1]);
+				var objectData = JSON.parse(response);
+				var hmUrls = objectData.health_monitor_refs;
+				var healthmonitorList= new Array();
+				for each(index in hmUrls){
+					var hmPath = index.split("/");
+					var hmResponse = aviPlugin.getObjectDataByUUID(hmPath[hmPath.length-2],hmPath[hmPath.length-1]);
```

Comment:
can you check the len of the path here?

Reply:
This is reference so it will always have objectType and uuid at that position.